sdk/python/jobs/automl-standalone-jobs/automl-forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1435 @@
+    "\n",
+    "pipeline_submitted_job_base = ml_client.jobs.create_or_update(\n",
+    "    pipeline_job_base,\n",
+    "    experiment_name=base_experiment_name,\n",
+    "    skip_validation=True,\n",
+    ")\n",
+    "ml_client.jobs.stream(pipeline_submitted_job_base.name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# # To rehydrate baseline run\n",
+    "# RUN_ID = <Paste the PipelineRunId from the output of the previous cell.>\n",
+    "# pipeline_submitted_job_base = ml_client.jobs.get(RUN_ID)\n",
+    "# pipeline_submitted_job_base"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 8. Download Pipeline Output\n",
+    "Next, we will download the output files generated by the compute metrics components for each executed pipeline and save them in the corresponfing subfolder of the `output` folder. First, we create corresponding output directories. Then, we execute the `ml_client.jobs.download` command which downloads experiments' outputs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create output directories\n",
+    "mm_output_dir = os.path.join(os.getcwd(), \"output/many-models\")\n",
+    "base_output_dir = os.path.join(os.getcwd(), \"output/base\")\n",
+    "\n",
+    "os.makedirs(mm_output_dir, exist_ok=True)\n",
+    "os.makedirs(base_output_dir, exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.download(\n",
+    "    pipeline_submitted_job.name, download_path=mm_output_dir, output_name=\"output_files\"\n",
+    ")\n",
+    "ml_client.jobs.download(\n",
+    "    pipeline_submitted_job_base.name,\n",
+    "    download_path=base_output_dir,\n",
+    "    output_name=\"output_files\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "## 9. Compare Evaluation Results\n",
+    "\n",
+    "### 9.1. Examine Metrics\n",
+    "\n",
+    "In this section, we compare metrics for the 2 pipeline runs to quantify accuracy improvement of AutoML over the baseline model. First, we compare metrics that are calculated for the entire dataset. Since there are 10 unique time series in the test dataset, these individual metrics are aggregated into a single number. The non-normalized metrics can be misleading because the difference in scales of each unique time series. The following [article (placeholder)](https://review.learn.microsoft.com/en-us/azure/machine-learning/how-to-understand-automated-ml?view=azureml-api-2&branch=pr-en-us-238443#forecasting-metrics-normalization-and-aggregation) explains this topic in a greater detail.\n",
```

Comment:
Do we need the word "(placeholder)"?

Reply:
This is just temporary. I need this for quick find and replacement once the documentation ships.